.gitignore

```diff
@@ -50,6 +50,7 @@ coverage.xml
 .hypothesis/
 test.env
 
+
```

Comment:
Nit: is there a reason for this line addition? If not, lets get rid of it

Reply:
change made locally will push up